client/src/app/core/ui-services/base-sort-list.service.ts

```diff
@@ -76,6 +77,24 @@ export abstract class BaseSortListService<V extends BaseViewModel> {
         return this.sortDefinition.sortAscending;
     }
 
+    /**
+     * Setting the sorting function
+     *
+     * @param sortFn The function to customize the sorting.
+     */
+    public set sortFn(sortFn: (itemA: V, itemB: V, ascending: boolean, intl?: Intl.Collator) => number) {
```

Comment:
The getter and setter are not necessary, because both are public. Then you just can make the member `_sortFn` public and rename it to `sortFn`.